vscode-wpilib/resources/webviews/projectcreator.html

```diff
@@ -19,7 +19,7 @@
 
   <h1>Welcome to WPILib New Project Creator</h1>
 
-  <button id="projectType" >Select a project type (Example or Template)</button>
+  <button id="projectType" >Select a project type (Example, Template or RobotBuilder)</button>

```

Comment:
You need an Oxford comma:
```suggestion
  <button id="projectType" >Select a project type (Example, Template, or RobotBuilder)</button>
```